lib/testing/src/lib/core/pages/form/widgets/displayTextWidget.ts

```diff
@@ -16,23 +16,23 @@
  */
 
 import { FormFields } from '../formFields';
-import { by } from 'protractor';
+import { by, Locator } from 'protractor';
 
 export class DisplayTextWidget {
 
-    formFields = new FormFields();
-    labelLocator = by.css('div[class*="adf-display-text-widget"]');
-    inputLocator = by.css('input');
+    formFields: FormFields = new FormFields();
+    labelLocator: Locator = by.css('div[class*="adf-display-text-widget"]');
+    inputLocator: Locator = by.css('input');
 
-    getFieldLabel(fieldId) {
+    getFieldLabel(fieldId): Promise<string> {
```

Comment:
async for code consistency with all other pages

Reply:
see above